west.yml

```diff
@@ -128,9 +130,10 @@ manifest:
     - name: mipi-sys-t
       path: modules/debug/mipi-sys-t
       revision: 75e671550ac1acb502f315fe4952514dc73f7bfb
-    - name: nrf_hw_models
+    - name: ext_NRF52_hw_models
```

Comment:
next time use `repo-path` so that you don't change the name and hence the bot can work

Reply:
changed to: https://github.com/zephyrproject-rtos/zephyr/blob/be2a233296f4496a2baf3b2bed2d48a0a840fc4a/west.yml
but now CI is broken, works locally.